js/values/syllableCountMethod.js

```diff
@@ -0,0 +1,53 @@
+var LanguageSyllableRegex = require ( "./syllableCountStep.js" );
+
+var isUndefined = require( "lodash/isUndefined" );
+var forEach = require ( "lodash/forEach" );
+
+/**
+ * Creates a Language Syllable Regex Master
+ *
+ * @param {object} config The config object containing an array with syllable exclusions
+ * @constructor
+ */
+var syllableCountIterator = function( config ) {
```

Comment:
I love `Iterator` as name for this but I prefer the filename to reflect this too.